test/controllers/categories_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class CategoriesControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get categories_index_url
+    assert_response :success
+  end
```

Comment:
We should have some tests which checks if we at least display names of categories.